doc-content/kie-server-rest-api-custom-queries-overview-ref.adoc

```diff
@@ -0,0 +1,18 @@
+[id='kie-server-rest-api-custom-query-endpoints-ref_{context}']
+= Custom query
+
+* *Custom queries:* Some custom query requests with the {KIE_SERVER} REST API require a query `mapper` definition to map the query results to concrete objects. You can implement your own query result mappers or use the mappers provided with {PRODUCT}. The query mappers in {PRODUCT} are similar to other object-relational mapping (ORM) providers, such as Hibernate, which maps tables to entities. For example, you can use the `org.jbpm.kie.services.impl.query.mapper.ProcessInstanceQueryMapper`, also registered as `ProcessInstances`, in custom query endpoints for returning process instance data.
```

Comment:
```suggestion
Some custom query requests with the {KIE_SERVER} REST API require a query `mapper` definition to map the query results to concrete objects. You can implement your own query result mappers or use the mappers provided with {PRODUCT}. The query mappers in {PRODUCT} are similar to other object-relational mappings (ORM) providers, such as Hibernate, which maps tables to entities. For example, you can use the `org.jbpm.kie.services.impl.query.mapper.ProcessInstanceQueryMapper`, also registered as `ProcessInstances`, in custom query endpoints for returning process instance data.
```

Reply:
This file was supposed to be deleted. Something went haywire, checking.